app/src/main.cpp

```diff
@@ -83,6 +101,44 @@ void loop() {
 
   monitorAudioEngine();
 
+  // Run display controls
+  disp.displayVol();
+  disp.displayPan();
+  disp.showLib();
+
+  // Get name change from library selection
+  if(disp.getNameChange()){
+    String name = disp.getFileName();
+    Serial.println(name);
+    track.setWriteFileName(name);
+    disp.setNameChange(false);
+  }
+
+  // Mointor mode change
+  if(disp.getModeChange() == 1){
+    if(disp.getMode() == 1){
+      mode = Mode::Replace;
+      disp.setModeChange(0);
+      Serial.println("Mode: REPLACE");
+    }
+    else{
+      mode = Mode::Overdub;
+      disp.setModeChange(0);
+      Serial.println("Mode: OVERDUB");
+    }
+  }
+  
+  // Moinitor reverse flag
+  if(disp.getRevBool()){
+    Serial.println("Reverse Entered");
+    //track.reverse();        Commented out - Need work
+    disp.setRevBool(false);
+    status = Status::Play;
+  }
```

Comment:
Until reverse is working, let's remove the related code from this pull request. It can be included in a separate PR.

Reply:
sounds good!